src/components/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faGithub,
+  faLinkedin,
+  faSafari,
+} from "@fortawesome/free-brands-svg-icons";
+import { Theme } from "../theme/theme";
+
+export const Footer = () => {
+  return (
+    <FooterDiv>
+      <pre>Made by Satellytes · </pre>
+      <SocialLink
+        title="Satellytes on LinkedIn"
+        href="https://www.linkedin.com/company/satellytes/"
+      >
+        <SocialIcon icon={faLinkedin} />
+      </SocialLink>
+      <SocialLink
+        title="BadgeForge on GitHub"
+        href="https://github.com/satellytes/badge-forge"
+      >
+        <SocialIcon color={Theme.colors.gray300} icon={faGithub} />
+      </SocialLink>
+      <SocialLink title="Satellytes Website" href="https:/satellytes.com/">
+        <SocialIcon color={Theme.colors.gray300} icon={faSafari} />
+      </SocialLink>
+    </FooterDiv>
+  );
+};
+
+const FooterDiv = styled.div`
```

Comment:
```suggestion
const Wrapper = styled.footer`
```

I'd suggest we use a `footer` element instead of the generic `div`.
Since renaming the `FooterDiv` to `Footer` would collide with the component itself, I suggested using the generic name `Wrapper`.